c7n/resources/rdsparamgroup.py

```diff
@@ -0,0 +1,339 @@
+# Copyright 2017 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import itertools
+import logging
+
+from botocore.exceptions import ClientError
+
+from c7n.actions import ActionRegistry, BaseAction
+from c7n.filters import FilterRegistry
+from c7n.manager import resources
+from c7n.query import QueryResourceManager
+from c7n.utils import (type_schema, local_session)
+
+log = logging.getLogger('custodian.rds-param-group')
+
+pg_filters = FilterRegistry('rds-param-groups.filters')
```

Comment:
this should match the resource type name i think (note pluralization). also you don't have to specify these out of band, the metaclass will auto create them for you though its more typing with the resource manager class name as a prefix.